### ISSUE TITLE
'yarp name get /port accepts' does NOT return anything

### ISSUE THREAD

**Issue by aikolina** [robotology/yarp#1197]
Hello, 

I've just started to follow the tutorial on YARP ([Getting started with YARP ports](http://www.yarp.it/note_ports.html)) and I had a little problem while testing the command `yarp name get /port accepts` on Ubuntu 16.04.

According to the tutorial, the result of the command should be the following one:
```
[you type] yarp name get /summer accepts
[response] port /summer property accepts = tcp text text_ack udp mcast shmem
           *** end of message
```
On the contrary, in my case I get nothing:
```
adinale@iitrbcslap125:~$ yarp name get /summer accepts
port /summer property accepts =
*** end of message
```
But apparently, other people who has tested this command got at least a partial results like:
```
port /summer property accepts = udp mcast
*** end of message
```
So I have few questions regarding this issue:
- Is this a common misbehavior?
- Do I need some specific setup to make this command work?
- Is there an updated version of this command that I should use?

Thanks!

**NOTE:** I would like to underline that the ports that I created before reaching this critical point of the tutorial work perfectly!

**Comment by drdanz** [comment]
@aikolina Are you using `master` or `devel`

**Comment by aikolina** [comment]
Actually, I installed YARP from the binaries. So probably it's not the `devel` branch.

**Comment by traversaro** [comment]
On the latest `devel` on Ubuntu 16.04 I also get the same empty list as result of the `accept` command, opening two ports using `yarp read` and `yarp write` .

**Comment by drdanz** [comment]
After some more debugging, I found out that everything seems to be working, excepts for the part that inserts the property "accepts" into yarpserver database.
Executing ``yarp name set /summer accepts tcp text text_ack udp mcast shmem`` sets this property, and therefore any following ``yarp name get /summer accepts`` returns the expected output.

Somewhere hidden inside YARP there should have been some place where this `set` command was executed.

**Comment by drdanz** [comment]
@Nicogene found out that in `NameClient.cpp` this part was commented out.

The responsible commit seems to be 02f9b1983614ef08a2ada598a38a357e68723731

**Comment by Nicogene** [comment]
Fixed by #1249

**Comment by drdanz** [comment]
I was right, it was already there in YARP 2.3.20... actually it was there in since v2.3.7 :+1:
